test/ngMock/angular-mocksSpec.js

```diff
@@ -351,6 +351,18 @@ describe('ngMock', function() {
     }));
 
 
+    it('should allow you to NOT specify the delay time', inject(function($interval) {
+      var counter = 0;
+      $interval(function() {counter++;});
+      $interval(function() {counter++;}, 0);
+
+      $interval.flush(1000);
+      expect(counter).toBe(2000);
```

Comment:
When calling interval it checks the second argument to be a number or equivalent to zero, hence undefined gives the same results. Since above two statements are increasing counter on a flush of 1000 both functions are run and counter finish with a value of 2000.